src/stopped.rs

```diff
@@ -108,34 +108,38 @@ impl StoppedJail {
 
         let mut params = self.params.clone();
 
-        // Set the IP Addresses
-        params.insert(
-            "ip4.addr".into(),
-            param::Value::Ipv4Addrs(
-                self.ips
-                    .iter()
-                    .filter(|ip| ip.is_ipv4())
-                    .map(|ip| match ip {
-                        net::IpAddr::V4(ip4) => *ip4,
-                        _ => panic!("unreachable"),
-                    })
-                    .collect(),
-            ),
-        );
+        let ipv4_addresses: Vec<_> = self.ips
+            .iter()
+            .filter(|ip| ip.is_ipv4())
+            .map(|ip| match ip {
+                net::IpAddr::V4(ip4) => *ip4,
+                _ => panic!("unreachable"),
```

Comment:
you may wanna use [`unreachable!()`](https://doc.rust-lang.org/std/macro.unreachable.html) here

Reply:
Nice catch, sure!